app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/itemlisting/VaultItemListingScreen.kt

```diff
@@ -433,7 +434,8 @@ private fun VaultItemListingScaffold(
                 FloatingActionButton(
                     containerColor = MaterialTheme.colorScheme.primaryContainer,
                     onClick = vaultItemListingHandlers.addVaultItemClick,
-                    modifier = Modifier.testTag("AddItemButton"),
+                    modifier = Modifier.testTag("AddItemButton")
+                        .systemBarsPadding(),
```

Comment:
noticed the FAB was hiding behind the device OS navigation bar

Reply:
⛏️  Can we make this change in a separate PR since it's unrelated to the purpose of this one?

Reply:
I have additional concerns on this as well, since the scaffold should be managing this for us.

What scenario did you see this in?

Reply:
`NavGraphBuilder.vaultItemListingDestinationAsRoot` when the list is navigated to via this.

Reply:
Ahh, I see it now. I agree this could be a separate PR.

Additionally, I think we can fix this generically in the `BitwardenScaffold`:

```kotlin
        floatingActionButton = {
            Box(modifier = Modifier.navigationBarsPadding()) {
                floatingActionButton()
            }
        },
```

Reply:
created https://bitwarden.atlassian.net/browse/PM-10645 to capture this